fuzz/fuzzers/utils.rs

```diff
@@ -0,0 +1,48 @@
+extern crate base64;
+extern crate rand;
+extern crate ring;
+
+use self::base64::*;
+use self::rand::{Rng, SeedableRng, XorShiftRng};
+use self::rand::distributions::{IndependentSample, Range};
+use self::ring::digest;
+
+pub fn random_config(data: &[u8]) -> Config {
+    // use sha256 of data as rng seed so it's repeatable
+    let sha = digest::digest(&digest::SHA256, data);
```

Comment:
Could a more lightweight dependency like [sha2](https://github.com/RustCrypto/hashes/blob/457061f19b3538651aa7ef208ce6b04bfba65e61/sha2/Cargo.toml) be used here instead of `ring`?

Reply:
I don't see why not.